.circleci/config.yml

```diff
@@ -82,7 +110,23 @@ jobs:
     steps:
       - checkout
       - greetings-from-branch
-  
+  deploy:
```

Comment:
Deployment should be triggered only after you marge changes to the master branch.
You do not want to deploy changes from other branches

Reply:
I know, but i made it like this only to test it, i will add option for triggering only after merging to master :)